Gemfile

```diff
@@ -1,46 +1,38 @@
 source 'https://rubygems.org'
 
 gem 'sufia', '~> 7.2.0'
+# Hidden Sufia Dependencies
 gem 'flipflop', git: 'https://github.com/jcoyne/flipflop.git', branch: 'hydra'
+gem 'jbuilder', '~> 2.0' # Hidden Sufia Dependencies...Need PR against sufia...
+# gem 'rsolr', '~> 1.0'
+gem 'devise'
+# gem 'devise-guests', '~> 0.3'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 4.2.0'
+gem 'rails', '~> 5.0.0', '>= 5.0.0.1'
 # Use sqlite3 as the database for Active Record
 gem 'mysql2'
 # Use SCSS for stylesheets
 gem 'sass-rails', '~> 5.0'
+# Use Puma as the app server
+gem 'puma', '~> 3.0'
```

Comment:
Puma was brought into Rails 5 for actioncable. So guess we don't really need this as I didn't bring in actioncable with the upgrade. So if you guys dislike puma we can remove this.